05-read-write-plot.Rmd

```diff
@@ -421,22 +419,22 @@ saveWidget(mapview_obj, file="my_interactive_map.html")
 
 1. List and describe three types of vector, raster, and geodatabase formats.
 
-1. Name at least differences between `read_sf()` and the more well-known function `st_read()`.
+2. Name at least two differences between `read_sf()` and the more well-known function `st_read()`.
 
-1. Read the `cycle_hire_xy.csv` file from the **spData** package (Hint: it is located in the `misc\` folder).
-What is a geometry type of a new object? 
+3. Read the `cycle_hire_xy.csv` file from the **spData** package (Hint: it is located in the `misc\` folder).
+What is a geometry type of the loaded object? 
 
-1. Download Germany borders data using **rnaturalearth** and create a new object called `germany_borders`.
+4. Download the borders of Germany using **rnaturalearth**, and create a new object called `germany_borders`.
 Write this new object to a file of the GeoPackage format.
 
-1. Download raster data of global monthly minimum temperature with five minutes resolution using the **raster** package.
-Extract only values for June and save them to the `tmin_june.tif` file (Hint: use `raster::subset()`).
+5. Download the global monthly minimum temperature with a temporal resolution of five minutes using the **raster** package.
+Extract the June values, and save them to a file named `tmin_june.tif` file (Hint: use `raster::subset()`).
 
-1. Create a static map of Germany borders and save it to a PNG file.
+6. Create a static map of Germany's borders, and save it to a PNG file.
 
-1. Create an interactive map using data from the `cycle_hire_xy.csv` file. 
-Export this map to the HTML file called `cycle_hire.html`.
+7. Create an interactive map using data from the `cycle_hire_xy.csv` file. 
+Export this map to a file called `cycle_hire.html`.
 
-[^1]: A list of supported vector formats and theirs options could be found at http://www.gdal.org/ogr_formats.html.
+[^1]: A list of supported vector formats and theirs options can be found at http://www.gdal.org/ogr_formats.html.
```

Comment:
Yes this is definitely right - alongside your other changes these are great improvements.